templates/company.json

```diff
@@ -71,14 +100,25 @@
               "add",
               "_id",
               [
-                "string",
-                "_S.objectId"
+                "coalesce",
```

Comment:
Think you need to wrap the parameters to the "coalesce" function in a list (here and elsewhere changed).